src/custom/pages/Profile/index.tsx

```diff
@@ -216,13 +228,22 @@ export default function Profile() {
             </span>
           </BalanceDisplay>
           <CardActions>
-            <ExtLink href={getBlockExplorerUrl(chainId || 1, COW_CONTRACT_ADDRESS[chainId || 1], 'address')}>
-              {chainId === ChainId.XDAI ? 'Blockscout' : 'Etherscan'} ↗
+            <ExtLink
+              title="View contract"
+              href={getBlockExplorerUrl(chainId, COW_CONTRACT_ADDRESS[chainId], 'address')}
+            >
+              Contract ↗
```

Comment:
More than contract, could be View Token and send you to Etherscan Token page?

Reply:
i.e. 

https://etherscan.io/token/0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB

https://blockscout.com/xdai/mainnet/token/0x177127622c4A00F3d409B75571e12cB3c8973d3c/token-transfers

Reply:
Yes that might be a better destination. Agree `View Token` looks more clear. Will check how it fits with the other labels.